composer-manifest.yaml

```diff
@@ -0,0 +1,204 @@
+packages:
```

Comment:
We don't want to include this file from the get-go.  It will be added to the derivative site on first composer run.

Reply:
This file was included so I could test the caching in the end to tests I was doing for the circleci jobs.  I'll remove it.